apps/shared-components/badge/badge.component.scss

```diff
@@ -16,6 +16,14 @@ div {
         font-size: 16px;
       }
     }
+
+    &.mini-text {
+      @apply com-py-2px com-px-1 com-leading-3 com-text-8px com-font-medium;
+
+      nb-icon {
+        @apply com-text-sm;
+      }
+    }
   }
```

Comment:
let's make mini to extra-small or xs

Reply:
extra small is already made with 10px font size, its 8px font size